README.md

```diff
@@ -26,19 +26,21 @@ BANNED PACKAGES
 * Install Postman https://www.getpostman.com/
 * Install gcc (windows https://sourceforge.net/projects/mingw-w64/?source=typ_redirect)
 * Go to api/ and create 3 folders bin, pkg and src
-* Run ''' go get github.com/gin-gonic/gin '''
-* Run ''' go get github.com/jinzhu/gorm '''
-* Run ''' go get github.com/mattn/go-sqlite3 '''
-* Run ''' go install github.com/mattn/go-sqlite3 // need gcc'''
-* Run ''' go build '''
-* Run ''' .\api.exe '''
+* Run ``` go get github.com/gin-gonic/gin ```
+* Run ``` go get github.com/jinzhu/gorm ```
+* Run ``` go get github.com/mattn/go-sqlite3 ```
+* Run ``` go install github.com/mattn/go-sqlite3 // need gcc ```
+* Run ``` go build ```
+* Run ``` .\api.exe ```
```

Comment:
This seems to be powershell specific, maybe specify how to on different platforms?